### PR TITLE
pcp-516

### PR DESCRIPTION
**Pull request by resmaa**
**Release 3.1**

Add a provision to configure Azure Storage Account from UI for images.

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1040#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/01-public-cloud/03-azure/02-create-azure-cluster.md
Comment (karl-cardenas-coding) [reply content/docs/04-clusters/01-public-cloud/03-azure/02-create-azure-cluster.md:162]: @resmaa it should be VNet ☝🏻

File: content/docs/04-clusters/01-public-cloud/03-azure/02-create-azure-cluster.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/03-azure/02-create-azure-cluster.md:141]: I'm not sure having another heading here is needed. We can include under Use Cases so that it's all together.

File: content/docs/04-clusters/01-public-cloud/03-azure/02-create-azure-cluster.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/03-azure/02-create-azure-cluster.md:135]:
```suggestion
You want to have a custom storage account for the following reasons:

```
Transitioned from passive to active voice.

File: content/docs/04-clusters/01-public-cloud/03-azure/02-create-azure-cluster.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/03-azure/02-create-azure-cluster.md:143]:
```suggestion
The following are steps to address common scenarios related to Azure cluster storage.
```
This suggestion avoids backtracking the reader by using "above."

File: content/docs/04-clusters/01-public-cloud/03-azure/02-create-azure-cluster.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/03-azure/02-create-azure-cluster.md:145]:
```suggestion
**Usecase: Default Storage Account and Storage Containers:**

```
We can do without the numbers but let's spell out usecase.

File: content/docs/04-clusters/01-public-cloud/03-azure/02-create-azure-cluster.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/03-azure/02-create-azure-cluster.md:150]:
```suggestion
**Usecase: Provide custom storage account and storage container**

```

File: content/docs/04-clusters/01-public-cloud/03-azure/02-create-azure-cluster.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/03-azure/02-create-azure-cluster.md:155]:
```suggestion
**Usecase: Custom Azure storage account**

```

File: content/docs/04-clusters/01-public-cloud/03-azure/02-create-azure-cluster.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/03-azure/02-create-azure-cluster.md:161]: @resmaa this is a security recommendation so let's use the warning box.

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
Feedback and suggestions below.

File: content/docs/04-clusters/01-public-cloud/03-azure/02-create-azure-cluster.md
Comment (ritawatson) [inline content/docs/04-clusters/01-public-cloud/03-azure/02-create-azure-cluster.md:145]: Should Usecase be two words? **Use Case**

**Review by ritawatson — APPROVED** [review]
Usecase should be two words, I think.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1040#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1040#pullrequestreview-1221252219" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-1/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@resmaa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | d7caee8fca0177fb6ece276aeccd7f610ac93017 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1040/librarium_build/1605977991647596544) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by karl-cardenas-coding** [comment]
Closing this PR due to merge conflicts related to the restructuring.  See PR #1062

**Pull request closed without merging**
